payment-cards/pci-compliance/saqs.md

```diff
@@ -4,24 +4,34 @@ title: Self Assessment Questionnaires
 permalink: /payment-cards/pci-compliance/saqs/
 ---
 
-To be [PCI Compliant](/payment-cards/pci-compliance/) as a merchant, you'll need to fill out an SAQ, or Self Assessment Questionnaire.
+As a merchant, the least burdensome way to become [PCI Compliant](/payment-cards/pci-compliance/) is to fill out a Self Assessment Questionnaire (the next step up would be to hire a [QSA](/glossary/#qsa)).
 
 Most likely, the 2 SAQ's that are relevant to you if you accept payments online are [SAQ A](#saq-a) and [SAQ A-EP](#saq-a-ep). For a full list, see [Understanding the SAQs for PCI DSS version 3](https://www.pcisecuritystandards.org/documents/Understanding_SAQs_PCI_DSS_v3.pdf).
 
-Following SAQ rules is not enough to ensure a secure payments flow. Keeping up with industry-standards is important to make sure payment information stays secure.
+<div class="popout warning">
+Following SAQ rules is not enough to ensure a secure payments flow. Keeping up with industry standards is important to make sure payment information stays secure.
+</div>
 
 ## SAQ A
 
-The easiest. Requires use of an iframe or a redirect so that payment information is entered directly on the [provider's](/glossary/#provider) domain.
+This level is designed to be easy to integrate and to expose the merchant to as little fraud risk as possible. SAQ A requires use of an iframe or a redirect so that payment information is entered directly on a [PCI Level 1](/payment-cards/pci-compliance/levels/) compliant domain, like a [payment gateway](/glossary/#gateway).
```

Comment:
Ah, thanks for catching that! That's what I had thought but I'm not sure I have anything to back it up. The exact wording from the [SAQ A AOC](https://www.pcisecuritystandards.org/documents/SAQ_A_v3.pdf) is:

> "All payment acceptance and processing are entirely outsourced to PCI DSS validated third-party service providers"

I'll update it to be something like:

> SAQ A requires use of an iframe or a redirect so that payment information is entered directly on a PCI DSS validated third-party domain, like a payment gateway.